### DISCUSSION TITLE
How to setup dev environment

### DISCUSSION THREAD

**Discussion by alan-palacios** [mbnuqw/sidebery#971] [Q&A]
Hi! I've been using sidebery and it's great. Unfortunately I have some issues where new tabs are being placed in the wrong panel.
I want to check what is the problem myself but had some problems running the browser with addon as there isn't a webext.run.mjs script as declared in the package.json: `"dev.run": "node build/webext.run.mjs --dev",` and it's not being generated with the `npm run dev` command neither.
I'm missing some previous setup?

**Comment by mbnuqw** [comment]
Sorry, I forgot to update .gitignore file, now fixed.
